gateway/index.js

```diff
@@ -21,11 +26,33 @@ const serviceList = [
   const schema = getStitchedSchemaFromSupergraphSdl({
     supergraphSdl,
   });
+
   const server = new ApolloServer({
     schema,
   });
 
-  server.listen(4000).then(({ url }) => {
-    console.log(`🚀 Server ready at ${url}`);
+  if (process.env.GQL_TRACING) {
```

Comment:
Shouldn't it be like this?
```suggestion
  if (process.env.GQL_TRACING === 'true') {
```

Reply:
@adijesori seems to be working without, like in your real code

Reply:
yes, but now that you put `false` in GW, it will still print it. No?